backend/src/services/cloudinary/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as cloudinary } from './cloudinary';
+export * from './helpers';
```

Comment:
We dont want to export cloudinary helpers as they are used only inside cloudinary service